sdks/python/.pylintrc

```diff
@@ -107,10 +110,9 @@ disable =
   no-self-use,
   no-value-for-parameter,
   not-callable,
-  # Re-enable the context manager check once https://github.com/PyCQA/pylint/issues/782 is fixed
-  not-context-manager,
```

Comment:
This is because the new version of pylint fixed this, right?

Reply:
No.  It is because the issue in question was closed and testing showed that I could just follow the instructions in the comment.